Foundation/NSGeometry.swift

```diff
@@ -9,10 +9,27 @@
 
 #if os(OSX) || os(iOS)
     import Darwin
-#elseif os(Linux)
+#elseif os(Linux) || os(Android)
     import Glibc
 #endif
 
+#if os(Android)
+private func floor( value: CGFloat.NativeType ) -> Double {
+    return floor( Double(value) )
+}
+private func ceil( value: CGFloat.NativeType ) -> Double {
+    return ceil( Double(value) )
+}
+private func round( value: CGFloat.NativeType ) -> Double {
+    return round( Double(value) )
+}
+#if false
```

Comment:
This and the other changes to CGFloat implementations aren't very clear to me. CGFloat should actually be Float on 32-bit and Double on 64-bit.